source/views/directory/list.tsx

```diff
@@ -47,14 +87,16 @@ export function DirectoryView(): JSX.Element {
 		isPending,
 		isInitial,
 		isLoading,
-	} = useDirectory(searchQuery)
+	} = useDirectory(searchQuery, searchQueryType)
 
 	React.useLayoutEffect(() => {
 		navigation.setOptions({
 			headerSearchBarOptions: {
 				barTintColor: c.white,
-				onChangeText: (event: ChangeTextEvent) =>
-					setTypedQuery(event.nativeEvent.text),
+				onChangeText: (event: ChangeTextEvent) => {
+					setSearchQueryType('Query')
```

Comment:
What happens visually if you're looking at some department results, and start typing?

Reply:
It behaves the same as searching a Query -- no difference in that UI case.

Reply:
I have a more in-depth answer:

1. Search > result detail > tap `Department` > `Query` search
    - Initially (debounced by 500ms):
      - Results replace placeholder screen
      - Navigation is unchanged
    - Search (debounced by 500ms):
      - Old results are replaced by new results
      - Navigation is unchanged
    - Tapping cancel:
       - Results are cleared
       - Navigation stack is intact -- are we wanting to cache results to restore them in this case?

Reply:
@hawkrives let me know if there's anything left to change here, otherwise we can tackle additions in a follow-up.